security/pkg/nodeagent/caclient/providers/citadel/client.go

```diff
@@ -46,6 +46,11 @@ var (
 	// with extra SAN (labels, etc) in data path.
 	ProvCert = env.RegisterStringVar("PROV_CERT", "",
 		"Set to a directory containing provisioned certs, for VMs").Get()
+
+	// OutputKeyCertToDir path is set, it will restore the cert from the signed by the CA
+	OutputKeyCertToDir = env.RegisterStringVar("OUTPUT_CERTS", "",
```

Comment:
It seems this is used to `Load the certificate from disk`, why it's called OUTPUT?

Reply:
same function written by former people, and there are lots of function used it to store the secret after getting the response 
1.https://github.com/istio/istio/blob/ac15f04154b180fb470722265baf3d05437b19c3/pilot/cmd/pilot-agent/main.go#L99 
2.https://github.com/istio/istio/blob/9c0e6e843cd30be9ffd69de0991b91e37f60144e/security/pkg/nodeagent/cache/secretcache.go#L685

Reply:
I think it makes sense in those places because it is used as the place to **write** the generated certificates, but here I guess we only **read** it and not write, do we need to stick to the name in order to avoid some config change?

Reply:
Let's not overload: this serves the purpose of saving the generated certs. It is called output because it writes, and there is no reason to confuse people by renaming it and making it do something else. 

If you need an env variable to indicate where to read - add one.

Reply:
Except we already have 2 already for that - the PROV_CERT indicates where to read long-lived certs, and /etc/certs continues to be used. Not clear if 3 setting for 'where to read some certs' are really needed.